src/sailsSocket.js

```diff
@@ -70,14 +70,19 @@ angular.module('bethel.sailsSocket', []).provider('sailsSocket', function() {
       return found;
     }
 
+    function handleError(rejectPromise, response, data) {
+      $rootScope.$broadcast('sailsSocket:error', response, data);
```

Comment:
For consistency can we have the broadcast order the parameters with `data` first and `response` second?